pages/trust-admin.js

```diff
@@ -145,9 +146,13 @@ const TrustAdmin = ({
               </div>
             </GridColumn>
           </GridRow>
-          {wardVisitTotalsStartDate && (
+          {(wardVisitTotalsStartDate || reportingStartDate) && (
             <ReviewDate>
-              Reporting for booked visits start date: {wardVisitTotalsStartDate}
+              {wardVisitTotalsStartDate &&
+                `Start date for booked visits reporting: ${wardVisitTotalsStartDate}`}
+              {wardVisitTotalsStartDate && reportingStartDate && <br />}
+              {reportingStartDate &&
+                `Start date for other reporting: ${reportingStartDate}`}
```

Comment:
A bit grim with all the conditionals. Any thoughts on refactoring?

Reply:
ReviewDate

Reply:
What if we used a ReviewDate component for each date and let the ReviewDate component handle the case where the date is null